client/verta/verta/operations/monitoring/alert/_alerter.py

```diff
@@ -16,41 +16,36 @@ class _Alerter(object):
 
     _TYPE = _AlertService.AlerterTypeEnum.UNKNOWN
 
-    def __init__(self, comparison):
-        if not isinstance(comparison, comparison_module._VertaComparison):
-            raise TypeError(
-                "`comparison` must be an object from verta.common.comparison,"
-                " not {}".format(type(comparison))
-            )
-
-        self._comparison = comparison
-
     def __repr__(self):
         return "<{} alert>".format(
             _AlertService.AlerterTypeEnum.AlerterType.Name(self._TYPE).lower()
         )
 
-    @property
-    def comparison(self):
-        return self._comparison
-
     @abc.abstractmethod
     def _as_proto(self):
         raise NotImplementedError
 
     @staticmethod
     def _from_proto(msg):
-        comparison = comparison_module._VertaComparison._from_proto(
-            msg.operator,
-            msg.threshold,
-        )
+        _protos_to_classes = {
+            _AlertService.AlertFixed: FixedAlerter,
+            _AlertService.AlertReference: ReferenceAlerter,
+            _AlertService.AlertRange: RangeAlerter,
+        }
+        if type(msg) in _protos_to_classes:
+            return _protos_to_classes[type(msg)]._from_proto(msg)
```

Comment:
I don't feel too strongly about this, but it _might_ be a good idea for the subclasses' method to be called something different. Elsewhere, I had spent more time than I'd like debugging an infinite recursion error because I was implementing a subclass but didn't realize I needed to override `_from_proto()` (since the base class implemented it).

But it would probably be the same time spent either way 🤷🏼‍♂️

```suggestion
            return _protos_to_classes[type(msg)]._from_proto_inner(msg)
```